.vscode/settings.json

```diff
@@ -6,7 +6,10 @@
   "files.trimTrailingWhitespace": true,
```

Comment:
not sure if you intended to make changes to this file?

Reply:
Yeh I was getting this weird thing where my VScode was adding a '?' to the beginning of long lines in metadata.yml files, and this seemed to be the only way to stop it. I am pretty sure that's the only change and then the rest of the diff is slight formatting changes.